roles/addons/diskless/files/disklessset.py

```diff
@@ -315,13 +351,15 @@ def generate_ipxe_boot_file(image_type, image_name, image_kernel, image_initramf
             print('SSH public key not found: {0}'.format(selected_ssh_pub_key))
             exit(1)
 
+        selinux_answer = str(input("Do you want to activate SELInux inside image? Enter yes or no: ").lower().strip())
```

Comment:
As per my previous comments, you should define a boolean from user answer. (if selinux_answer in ['yes', 'y']: selinux=True else: selinux=False)